e2e/cypress/integration/messaging/markdown_quotation_paragraphs_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18703-Markdown quotation paragraphs', () => {
+        const messageParts = ['this is', 'really', 'three quote lines'];
+        const message = `>${messageParts[0]}\n\n>${messageParts[1]}\n\n>${messageParts[2]}`;
+
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Post message to use
+        cy.postMessage(message);
+
+        var firstPartLeft;
+        cy.getLastPostId().then((postId) => {
+            // * There is only one blockquote, and therefore only one Quote icon
+            cy.get(`#postMessageText_${postId} > blockquote`).should('have.length', 1);
+
+            // * There are three distinct paragraphs, and therefore the three of them are separated by a space
+            cy.get(`#postMessageText_${postId} > blockquote > p`).should('have.length', 3);
```

Comment:
For some reason, this currently failed on me, probably related to recent Cypress upgrade.  Could please check again?  Thanks!

Reply:
Apparently, line breaks on 3.4 where considered as line breaks and on 3.6 as hitting on enter. Therefore, I had to change them so I use the {shift}{enter} way.

Also, clear is working fine on 3.6. Therefore, probably was not a problem with the platform but the version of Cypress.

Thanks!

Reply:
Thanks for checking!